app/controllers/recrypt/migrate_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
+#  Cryptopus and licensed under the Affero General Public License version 3 or later.
+#  See the COPYING file at the top-level directory or at
+#  https://github.com/puzzle/cryptopus.
+
+class Recrypt::MigrateController < RecryptController
```

Comment:
rename to
Recrypt::Sso ? /recrypt/sso

create
Recrypt::Ldap ? /recrypt/ldap

create two independent policies ?